linchpin/api/__init__.py

```diff
@@ -37,25 +86,19 @@ def run_playbook(self, pinfile, targets='all', playbook='provision'):
         """
 
         pf = yaml2json(pinfile)
-
```

Comment:
Nitpick here. Removing the spacing makes it harder to read this code. I specifically separated these lines for two reasons. 1) readability and 2) logical sectioning. Can we put the spacing back where appropriate?